src/Fabulous.Avalonia/ThemeAware.fs

```diff
@@ -1,42 +0,0 @@
-namespace Fabulous.Avalonia
-
-open Avalonia
-open Avalonia.Styling
-open Fabulous
-open Fabulous.Avalonia
-
-[<AbstractClass; Sealed>]
-type ThemeAware =
-    static member With(light: 'T, dark: 'T) =
-        if Application.Current.ActualThemeVariant = ThemeVariant.Dark then
-            dark
-        else
-            light
```

Comment:
@edgarfgp Will this helper be dropped without replacement or is there another mechanism I can use to switch between dark an light widget colors without an `if` statement?

Reply:
We have added support for `Environment` objects similar t SwiftUI. See https://github.com/fabulous-dev/Fabulous.Avalonia/blob/340c3cbc446c23a260b7598533410749ace0adbf/samples/Gallery/Pages/ThemeAwarePage.fs#L64 for an example